Mathlib/Data/Finset/Basic.lean

```diff
@@ -2618,7 +2620,6 @@ theorem filter_comm (s : Finset α) : (s.filter p).filter q = (s.filter q).filte
 #align finset.filter_comm Finset.filter_comm
 
 -- We can simplify an application of filter where the decidability is inferred in "the wrong way"
-@[simp]
```

Comment:
I've forgotten why/when this was removed.

Reply:
It does seem excessively opinionated as a simp lemma, though. I'm happy it's removed. If someone is excited to try backporting to master?

Reply:
https://github.com/leanprover-community/mathlib4/pull/12576